src/otx/recipe/classification/h_label_cls/deit_tiny.yaml

```diff
@@ -3,16 +3,6 @@ model:
   init_args:
     arch: "deit-tiny"
 
-    multiclass_loss_callable:
-      class_path: torch.nn.CrossEntropyLoss
```

Comment:
@harimkang
Why is it `reduction="mean"` (default of `torch.nn.CrossEntropyLoss`) although multiclass cls model has `reduction="none"`.

Reply:
Um I remember it being imported as it was used in MM, some calculations were used differently in head.